app/connector/servicenow/src/main/java/io/syndesis/connector/servicenow/ServiceNowMetadataRetrieval.java

```diff
@@ -34,16 +34,14 @@
 import org.apache.camel.CamelContext;
 import org.apache.camel.component.extension.MetaDataExtension;
 import org.apache.camel.component.servicenow.ServiceNowConstants;
+import org.apache.camel.component.servicenow.ServiceNowMetaDataExtensionWrapper;
 import org.apache.camel.util.ObjectHelper;
 
 public final class ServiceNowMetadataRetrieval extends ComponentMetadataRetrieval {
```

Comment:
Is this class still needed ?

Reply:
The constructors for the `ServiceNowMetaDataExtension` are protected so I need to create them from somewhere inside the same package.

This was the quickest/cleanest solution I could think of for that. If you know a better approach, I'm all ears.